packages/components/src/site-thumbnail/index.tsx

```diff
@@ -24,11 +26,16 @@ export const SiteThumbnail = ( {
 	children,
 	alt,
 	mShotsUrl = '',
+	bgColorImgUrl,
 	size = 'small',
 	mshotsOption = MSHOTS_OPTION,
 }: Props ) => {
 	const { src, isLoading, isError, imgRef } = useMshotsImg( mShotsUrl, mshotsOption );
 
+	const primary = useDominantColor( bgColorImgUrl );
+
+	backgroundColor = primary?.hexa || backgroundColor;
```

Comment:
Let's try not to reassign the props object. React might get confused with the references when diffing and it might bring us unexpected behaviors.

I suggest creating a new variable and using the `backgroundColor` variable inside it.

Reply:
Nice catch!